source/documentation/breaking-changes.html.md.erb

```diff
@@ -37,3 +37,6 @@ These breaking changes are coming soon or have recently been released:
 
 * [The syntax for CSS custom property values changed](breaking-changes/css-vars)
   in Dart Sass 1.0.0, LibSass 3.5.0, and Ruby Sass 3.5.0.
+
+* [`random($limit)` with units](breaking-changes/random-with-units) beginning in
```

Comment:
To match the order of other changes here, more recent changes should go at the top.